mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -175,6 +183,8 @@ final class OneVsRestModel private[ml] (
         val updateUDF = udf { (predictions: Map[Int, Double], prediction: Vector) =>
           predictions + ((index, prediction(1)))
         }
+        model.setFeaturesCol($(featuresCol))
+        model.setPredictionCol($(predictionCol))
```

Comment:
It's not necessary to ```setPredictionCol``` here, since the prediction column of each base binary classifier is ignored during the transformation. The ```predictionCol``` of ```OneVsRestModel``` was set at L198.